chroot_init_script.sh

```diff
@@ -5,13 +5,13 @@ CHROOT_DIR="/tmp/chroot"
 
 # Create necessary directories
 echo "Creating directory structure in ${CHROOT_DIR}..."
-mkdir -p "$CHROOT_DIR"/{bin,lib/x86_64-linux-gnu,usr/lib/x86_64-linux-gnu,usr/bin,usr/sbin,etc/security}
+mkdir -p "$CHROOT_DIR"/{bin,lib/x86_64-linux-gnu,usr/lib/x86_64-linux-gnu,usr/bin,usr/sbin,etc/security,usr/lib}
 
-# List of esential binaries
-ESSENTIAL_BINARIES="/bin/bash /bin/apt /usr/bin/apt-get /usr/bin/apt-cache /usr/sbin/apt /usr/bin/timeout"
+# List of essential binaries
+ESSENTIAL_BINARIES="/bin/bash /bin/apt /usr/bin/apt-get /usr/bin/apt-cache /usr/sbin/apt /usr/bin/timeout /bin/ls /usr/bin/python3  /usr/bin/python /usr/bin/env"
```

Comment:
As far as I understand this only handles latest python version downloaded by apt? We should consider how to enable different Python versions. I am not sure if python is backward compatible. In case it is, this comment should be ignored